pkg/deployments/deployments_test.go

```diff
@@ -71,6 +74,55 @@ func TestMakeControlGroup(test *testing.T) {
 	}
 }
 
+func TestDeploymentsFromConfig(test *testing.T) {
+	// create fake deployments file
+	data, err := json.MarshalIndent(
+		map[string]interface{}{
+			"v2": map[string]interface{}{
+				"deployments": map[string]interface{}{
+					"deploy.group": map[string]interface{}{
+						"docker_image": "image-abc123",
+						"git_sha":      "abc123",
+					},
+				},
+				"controls": map[string]interface{}{
+					"a_service:a_cluster.an_instance": map[string]interface{}{
+						"desired_state": "resurrection",
+						"force_bounce":  nil,
+					},
+				},
+			},
+		},
+		"",
+		"  ",
+	)
+	if err != nil {
+		test.Error("Failed to serialize test deployment JSON")
+	}
+
+	tempDir, err := ioutil.TempDir(os.TempDir(), "paasta-tools-go-test-service-*")
```

Comment:
I think you need to clean up after this

Reply:
I've added an `defer os.Remove` after this to clean up.